src/overlays/actors/ovl_En_Takaraya/z_en_takaraya.c

```diff
@@ -28,61 +46,390 @@ ActorInit En_Takaraya_InitVars = {
     (ActorFunc)EnTakaraya_Draw,
 };
 
-// static InitChainEntry sInitChain[] = {
-static InitChainEntry D_80ADFB20[] = {
+TexturePtr sEyesUpTextures[] = {
+    gTreasureChestShopGalEyeOpenUpTex,
+    gTreasureChestShopGalEyeHalfUpTex,
+    gTreasureChestShopGalEyeClosedTex,
+    gTreasureChestShopGalEyeHalfUpTex,
+};
+
+TexturePtr sEyesDownTextures[] = {
+    gTreasureChestShopGalEyeOpenDownTex,
+    gTreasureChestShopGalEyeHalfDownTex,
+    gTreasureChestShopGalEyeClosedTex,
+    gTreasureChestShopGalEyeHalfDownTex,
+};
+
+static InitChainEntry sInitChain[] = {
     ICHAIN_U8(targetMode, TARGET_MODE_6, ICHAIN_CONTINUE),
     ICHAIN_F32(targetArrowOffset, 1000, ICHAIN_STOP),
 };
 
-#endif
+u32 sTexturesDesegmented = false;
+
+u16 D_80ADFB2C[PLAYER_FORM_MAX] = {
+    1901, // PLAYER_FORM_FIERCE_DEITY
```

Comment:
This is an array of textIds, which we do in hex instead of dec
```suggestion
    0x76D, // PLAYER_FORM_FIERCE_DEITY
```